.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     - run: npm ci
     - run: npm test
     - run: npm run update && git diff --exit-code
-    - run: npm install --save-dev eslint@6 && npm test
+    - run: npm install --save-dev eslint@6 --save-dev eslint-plugin-unicorn@19 && npm run unit-test
```

Comment:
Do we need to install the old plugin version if we only run unit-test?

Reply:
Yes because the part that is failing is installation (due to the peer dependency issue).